src/main/java/com/example/betandwin/match/Match.java

```diff
@@ -0,0 +1,79 @@
+package com.example.betandwin.match;
+
+import com.example.betandwin.bet.Bet;
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+public class Match {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String host;
+    private String visitor;
+    private Integer hostScore;
+    private Integer visitorScore;
+
+    @Column(columnDefinition = "integer default 0")
+    private int betCount;
+
+    @OneToMany(mappedBy = "match", cascade = CascadeType.REMOVE)
+    private List<Bet> bets;
```

Comment:
Osobne liczenie zakładów i posiadanie kolekcji zakładów potencjalnie łatwo może doprowadzić do tego, że gdzieś zapomnimy zwiększyć, albo zmniejszyć liczbę zakładów dla danego meczu i rozmiar kolekcji może się rozjechać względem betCount. Na filmie pokazałem Ci jak mógłbyś to zrobić, żeby korzystać z rozmiaru kolekcji i wtedy usunąć pole betCount.
To trochę bardziej zaawansowane, więc polecam sobie przećwiczyć, ale punktów za to nie ucinam